Example/Pods/iOSPhotoEditor/Photo Editor/Photo Editor/PhotoEditor+Drawing.swift

```diff
@@ -1,80 +0,0 @@
-//
```

Comment:
Any word on this @jonchui ?

Reply:
This changes the example project to use Development Pods locally, 

which  allows you to make edits in the Example source code and seem them persisted right away, rather than editing a copy of the file (which is what pod install normally does) and then having to run `pod update` and changing the podspec every time

Does that make sense?

Check out `Using the files from a local path.` from https://guides.cocoapods.org/syntax/podfile.html#pod for more info